src/pzpr/env.js

```diff
@@ -61,18 +62,17 @@ pzpr.env = (function() {
 			isbrowser && document.createElement("a").download !== void 0
 	};
 
-	return {
+	var env = {
 		bz: bz,
 		OS: os,
 		API: api,
 		browser: isbrowser,
-		node: pzpr.Candle.env.node
+		node: Candle.env.node
 	};
-})();
 
-pzpr.lang = (function() {
-	var userlang = pzpr.env.node
-		? process.env.LANG
+	var userlang = env.node
+		? "XXX" // process.env.LANG
```

Comment:
This one I have no idea how to fix. There is no `process` variable when using `rollup`, which is something which comes from nodejs.